### PR TITLE
East sussex salesforce

### PR DESCRIPTION
**Pull request by struan**
Cobrand for back end only integration

[skip changelog]

### CODE REVIEW

File: perllib/FixMyStreet/Cobrand/EastSussex.pm [hunk 1]
```diff
@@ -6,4 +6,37 @@ use warnings;
...
+            $text .= "\n\n" . $field->{description} . "\n" . $q;
+            $row->detail($text);
+        }
+    }
+}
+
+sub open311_post_send {
+    my ($self, $row, $h, $contact) = @_;
+
+    $contact = $row->category_row;
```
Comment (chrismytton) [inline perllib/FixMyStreet/Cobrand/EastSussex.pm:27]: It's slightly confusing having `$contact` as an argument and then ignoring it but reusing the variable. Is there a reason for doing it like that?

File: web/cobrands/eastsussex/assets.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    read: function(json, type, filter) {
+        if (typeof json == "string") {
+            obj = OpenLayers.Format.JSON.prototype.read.apply(this,
+                                                              [json, filter]);
+        } else {
+            obj = json;
+        }
+
+      var results = [];
+      for (var i=0, len=obj.length; i<len; i++) {
```
Comment (chrismytton) [inline web/cobrands/eastsussex/assets.js:18]: Could use `$.each` here, but looking at the existing JS it seems that loops are done in both ways in lots of places. So not hugely important 🙂

File: web/cobrands/eastsussex/assets.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+
+OpenLayers.Format.EastSussex = OpenLayers.Class(OpenLayers.Format.JSON, {
+    read: function(json, type, filter) {
+        if (typeof json == "string") {
+            obj = OpenLayers.Format.JSON.prototype.read.apply(this,
+                                                              [json, filter]);
+        } else {
+            obj = json;
```
Comment (chrismytton) [inline web/cobrands/eastsussex/assets.js:14]: Doesn't look like this `obj` variable is defined with `var obj` anywhere?

**Comment by struan** [comment]
This has been merged as of 13e9cbb95d6ba619c4046bbbdcd7e7cb7b0f9e67

**Pull request closed without merging**